docs/users-guide/05-visualizing-results.md

```diff
@@ -140,13 +142,39 @@ If your table is a result that contains one numeric column and two grouping colu
 
 ![Pivot table](images/visualizations/pivot.png)
 
+This auto-pivoting is distinct from the pivot table visualization, which we cover next.
+
+### Pivot tables
+
+Pivot tables allow you swap rows and columns, group data, and include aggregate data in your table. You can group one or more metrics by one or more dimensions. 
+
+For a given query result, you can assign fields to one of three "buckets":
+
+- Fields to use for the table rows
+- Fields to use for the table columns
+- Fields to use for the table values 
+
+Let's say we ask the following question in the notebook editor:
+
+![Pivot table notebook](images/visualizations/pivot-table-notebook.png)
+
+From the `Orders` table, we've summarized the count of orders and the average order total, and grouped by `User->State`, `Product->Category`, and `Created At` binned by year. Here's our question visualized as a pivot table:
+
+![Pivot table options](images/visualizations/pivot-table-options.png)
+
+We've assigned the dimensions `User->State` and `Product-Category` to table rows, and assigned the dimension `Created At` as our columns: the years 2016, 2017, and so on. We can drag and drop dimensions between the row and column buckets, and add aggregations to the table values bucket. For example, if we assign a field to the columns bucket, Metabase will pivot that field and render each unique value of that field as a column heading.
+
+You can put multiple fields in each of these buckets, but note that the order of the fields changes how Metabase displays the table: each additional field will nest within the previous field. 
+
+Where it makes sense, Metabase will automatically include aggregations for grouped rows. For example, as in the image above, because we've grouped our rows first by `State`, then by `Product->Category`, Metabase will list each `Product->Category` (like Gizmo) for each `State`, and aggregate the metrics for that subgroup. For orders placed in Wisconsin, Metabase would sum the count of orders for each category, and find the average order total across all product categories. 
```

Comment:
```suggestion
Where it makes sense, Metabase will automatically include subtotals for grouped rows. For example, as in the image above, because we've grouped our rows first by `State`, then by `Product->Category`, Metabase will list each `Product->Category` (like Gizmo) for each `State`, and aggregate the metric(s) for that subgroup. For orders placed in Wisconsin, Metabase would sum the count of orders for each category, and find the average order total across all product categories in that state. 
```